src/PactNet/Drivers/IHttpInteractionDriver.cs

```diff
@@ -54,6 +54,15 @@ internal interface IHttpInteractionDriver : IProviderStateDriver
         /// </summary>
         /// <param name="contentType">Context type</param>
         /// <param name="body">Serialised body</param>
-        void WithResponseBody(string contentType, string body);
+        void WithResponseBody(string contentType, string body);
+
+

```

Comment:
Looks like an additional white space here - is there some common way to format C# code?